### ISSUE TITLE
[ nextcloud ] ResourceLocator can not find a web root (root: /apps2/deck, file: XXX.css)

### ISSUE THREAD

**Issue by clumsycat** [Wonderfall/dockerfiles#257]
description:
After i set some apps, the page is very odd, because browser can't load correct css file.
What can i do to fix this?

info in nextcloud.log
>
**DECK**
{"reqId":"7Xq1Rob9MvJIm7p0iM8V","level":3,"time":"2017-10-27 11:24:05","remoteAddr":"MyDockerIP","user":"XXXX","app":"lib","method":"GET","url":"\/apps\/deck\/","message":"**ResourceLocator can not find a web root (root: \/apps2\/deck, file: css\/deck\/bd5e80ef07099242f7d93f8b4a31507e-style.css, webRoot: , throw: true)**","userAgent":"Mozilla\/5.0 (Windows NT 6.1; WOW64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/61.0.3163.91 Safari\/537.36","version":"12.0.3.3"}
**contacts**
 {"reqId":"aqIfW2xO3acm083zzQlz","level":3,"time":"2017-10-27 11:43:25","remoteAddr":"MyDockerIP","user":"XXXXX","app":"lib","method":"GET","url":"\/apps\/contacts\/","message":"**ResourceLocator can not find a web root (root: \/apps2\/contacts, file: css\/contacts\/bd5e80ef07099242f7d93f8b4a31507e-style.css, webRoot: , throw: true)**","userAgent":"Mozilla\/5.0 (Windows NT 6.1; WOW64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/61.0.3163.91 Safari\/537.36","version":"12.0.3.3"}

**Comment by lachmanfrantisek** [comment]
There is a [bug in the server](https://github.com/nextcloud/server/issues/5289) caused by having a folder with apps outside of the root (like in this docker image). As a workaround, you can remove `scss` files from `css` folder for each app with problems. (Every app update will create them ones again.)

**Comment by skjnldsv** [comment]
Yeah, The server won't be able to fix this. This requires a nginx patch.
